apps/patient/src/components/LocationModal.tsx

```diff
@@ -80,16 +89,14 @@ export const LocationModal = ({ isOpen, onClose }: any) => {
   };
 
   const loadOptions = (inputValue: string, callback: (options: any) => void) => {
```

Comment:
This `loadOptions` function is just passing the variables to `debouncedSearchForLocations`, maybe get rid of it

Reply:
cleans up the jsx a little, but I guess it's a bit superfluous, I'll remove